### PR TITLE
Fix: bugfixes and overall improvements to alert history 

### PR DESCRIPTION
**Pull request by ahmadshaheer**
### Summary
- [x] 1. remove the severity label from the top level (this is redundant because you are already showing the severity with special colours)

Screenshot 2024-09-02 at 10 17 44 PM

- [x] 2. I find the alert number of redundant

Screenshot 2024-09-02 at 10 18 57 PM

- [x] 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert

Screenshot 2024-09-02 at 10 20 34 PM

- [x] 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.

after disable

Screenshot 2024-09-02 at 10 20 34 PM

after refresh

Screenshot 2024-09-02 at 10 24 44 PM

- [x] 6. make the default period 6h in history

- [x] 7. change the top contributors' description to
"Top contributors highlight the most frequently triggering group-by attributes in multi-dimensional alerts" because not every alert will need to have group-by.

- [ ] 8. Search doesn't work - we should either remove it or make it functional

- [x] 9. The timeline doesn't show all the items (I am not sure what the logic but it should show all the items in /overall_status)

- [x] 10. Let's not show the value column for now

- [x] 11. on the next page offset should be the total consumed so far but `1` is sent. It should be like 20, 40, etc..

- [x] 12. The logs link shows "View traces" and the traces link shows "View logs" link

#### Screenshots

NA

### CODE REVIEW

File: frontend/src/container/AlertHistory/Timeline/Table/Table.styles.scss [hunk 1]
```diff
@@ -88,6 +91,9 @@
 		}
 	}
 	.alert-history-label-search {
+		// .ant-select-selection-placeholder {
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Timeline/Table/Table.styles.scss:94]: Remove this

File: frontend/src/container/AlertHistory/Timeline/Table/Table.tsx [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-import { REACT_QUERY_KEY } from 'constants/reactQueryKeys';
+// import { initialFilters } from 'constants/queryBuilder';
 import {
 	useGetAlertRuleDetailsTimelineTable,
 	useTimelineTable,
 } from 'pages/AlertDetails/hooks';
-import { useMemo, useState } from 'react';
+import {
+	useMemo,
+	// , useState
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Timeline/Table/Table.tsx:11]: Remove commented code

File: frontend/src/container/AlertHistory/Timeline/Table/Table.styles.scss (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Timeline/Table/Table.styles.scss:94]: Done

File: frontend/src/container/AlertHistory/Timeline/Table/Table.tsx (hunk 2 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Timeline/Table/Table.tsx:11]: Done

**Comment by sharpshooter90** [comment]
@ahmadshaheer please got through the source design once, and reflect on the comments you have shared: https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2733-15237&t=ra6jnNGgu5FFNsFn-4

**Comment by sharpshooter90** [comment]
> 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert
A: add the description on hover, switch is clear enough to communicate what is it. History is inside the tab in lower level. There wont be any confusion. 

 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.
 A: @ahmadshaheer this is a functionality problem not a design issue.

File: frontend/src/container/AlertHistory/Timeline/Table/Table.tsx [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
 	const {
 		isLoading,
 		isRefetching,
 		isError,
 		data,
 		isValidRuleId,
 		ruleId,
-	} = useGetAlertRuleDetailsTimelineTable({ filters });
+	} = useGetAlertRuleDetailsTimelineTable();
+	// { filters }
```
Comment (SagarRajput-7) [inline frontend/src/container/AlertHistory/Timeline/Table/Table.tsx:22]: please make sure to remove all the unwanted comments once when the final PR against develop is ready

**Review by SagarRajput-7 — APPROVED** [review]

**Comment by ahmadshaheer** [comment]
> > 3. add a description to disable because it's not clear what the toggle does. does it turn off history or does it turn off/disable alert
> >    A: add the description on hover, switch is clear enough to communicate what is it. History is inside the tab in lower level. There wont be any confusion.
> 
> 4. The alert turn-off doesn't reflect immediately; needs a refresh. please rename the "turn-off" to "disable" for consistency.
>    A: @ahmadshaheer this is a functionality problem not a design issue.

@sharpshooter90 , 
> 3.
I have added a tooltip

> 4.

Yeah, I fixed this issue

File: frontend/src/container/AlertHistory/Timeline/Table/Table.tsx (hunk 3 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Timeline/Table/Table.tsx]: Done

**Review by YounixM — APPROVED** [review]

**Pull request merged**